wc/api.py

```diff
@@ -59,16 +59,23 @@ def __get_oauth_url(self, url, method):
 
         return oauth.get_oauth_url()
 
-    async def request(self, method, endpoint, data):
+    async def request(self, method, endpoint, data, ignore_headers):
```

Comment:
Depending on [this comment](https://github.com/channable/realtime/pull/323#discussion_r308098120), this might be changed to:
```python
async def request(self, method, endpoint, data, ignore_headers=False):
```

Reply:
Yeah, I like this better :).

Reply:
The user should only have to worry about the edge case whenever he actually needs too ;).

Reply:
yup, fixed